src/pgrn-custom-scan.c

```diff
@@ -392,6 +337,9 @@ PGrnSetRelPathlistHook(PlannerInfo *root,
 	CustomPath *cpath;
 	List *privateData = NIL;
 
+	if (!PGrnCustomScanInitialized)
+		return;
+
```

Comment:
Do we need this?
It seems that this isn't called after `PGrnFinalizeCustomScan()`.

Reply:
That’s true.